rpc/cloudagent/admin/logging/moc_cloudagent_logging.proto

```diff
@@ -26,3 +28,11 @@ message LogFileResponse {
 	string                      error    = 3;
 	string                      filename = 4;
 }
+
+message SetResponse {
+	bool                        done     = 1;	
+}
+
+message SetRequest {
+	uint32 settracelevel = 1;
```

Comment:
We need to take the location name in the input. We should be able to set the log level for all the node agents.